src/Ldap/Client.hs

```diff
@@ -211,7 +211,7 @@ output out conn = wrap . forever $ do
   msg <- atomically (readTQueue out)
   Conn.connectionPut conn (encode (toAsn1 msg))
  where
-  encode x = Asn1.encodeASN1' Asn1.DER (appEndo x [])
+  encode x = Asn1.encodeASN1' Asn1.BER (appEndo x [])
```

Comment:
I don't think this is necessary. Besides not being implemented in the `asn1-encoding` library, DER is a subset of BER, so this should not be a problem.